closed/custom/Images-pre.gmk

```diff
@@ -18,6 +18,20 @@
 # 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
 # ===========================================================================
 
+ifeq ($(OPENJDK_TARGET_OS), zos)
+	# Customized z/OS 31/64-bit Interoperability JNI header files.
+	# These files are generated as part of OpenJ9 build.
+	INCLUDE31_SOURCE_DIR := $(OPENJ9_VM_BUILD_DIR)/include31
+
+	# Stage into JDK base image 'include31' subdirectory
+	$(eval $(call SetupCopyFiles, JDK_COPY_INCLUDE31_HEADERS, \
+		SRC := $(INCLUDE31_SOURCE_DIR), \
+		FILES := $(call FindFiles, $(INCLUDE31_SOURCE_DIR)/), \
+		DEST := $(JDK_IMAGE_DIR)/include31/, \
+	))
+	JDK_TARGETS += $(JDK_COPY_INCLUDE31_HEADERS)
+endif
```

Comment:
These lines cannot begin with tabs - please use spaces.

Please remove the trailing slashes in the call to `FindFiles` and in the value of `DEST`, and add a comment on the `endif` line.